releng/travis-install.sh

```diff
@@ -18,6 +18,11 @@ GO_VERSION=1.13
 HELM_VERSION=2.9.1
 KUBECTL_VERSION=1.10.2
 KUBERNAUT_VERSION=2018.10.24-d46c1f1
+KIND_VERSION=v0.7.0  # see https://github.com/kubernetes-sigs/kind/releases
+
+# The ingress conformance tests (as a Zip, from GitHub)
+INGRESS_CONFORMANCE_TESTS_ZIP="https://github.com/alexgervais/ingress-controller-conformance/archive/ambassador-ref.zip"
```

Comment:
We should use a binary package as soon as it's available, perhaps from the task in this story.

Reply:
I agree. We could also build a Docker image with the Conformance tests and run them with a simple `docker run`. But any of these solutions would require some automatic build or CI on this repo, and that could take some time (and probably some discussions in the community).

Reply:
Submitted https://github.com/kubernetes-sigs/ingress-controller-conformance/pull/34 for packaging the conformance tests in a container.